e2e/bitrise.yml

```diff
@@ -0,0 +1,77 @@
+format_version: "11"
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+app:
+  envs:
+  - CACHE_KEY: gradle-configuration-cache-e2e-test
+
+workflows:
+  test_config_cache:
+    after_run:
+    - _setup
+    - _run
+    - _check
+
+  _run:
+    steps:
+    - path::./:
+        title: Execute step
+        run_if: "true"
+        is_skippable: false
+        inputs:
+        - verbose: true
+        - key_override: $CACHE_KEY
+
+  _check:
+    steps:
+#    - script: TODO after both steps have been merged
```

Comment:
Can this be removed?

Reply:
I'm not planning to leave this much longer and it should work so in theory we'll just have to uncomment when readding